pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_what-is-a-module.adoc

```diff
@@ -1,4 +1,3 @@
-include::../attributes.adoc[]
 [id='what-is-a-module_{context}']
 = What is a module
 
```

Comment:
Again, I can only add comment up to this line. Strange.

"A module, in modular documentation, is defined as topic-specific structured informational content written in the specified syntax, stored in a file, that can be published on its own and as a part of a larger document in {ProductShortName}."

 => 

"A module is a piece of structured content that can stand on its own or as part of a larger document. Generally, a module is stored in its own file. Asciidoc is the source language for modules in {ProductShortName}."

Reply:
"The following are the properties of a module:" => "A module has the following properties:"

- Self-contained and standalone look redundant to me.
- In Self-contained, you switch from talking about the module to talking about the user. Also uses passive. Feels strange. I suggest you change to: "A module contains all the necessary information that a user needs to achieve a goal. A module explains a concept, provides a procedure to perform a task, or presents reference information."